client/src/components/relatedProducts/YourOutfitAdder.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import EachOutfit from './EachOutfit.jsx';
+
+class YourOutfitAdder extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      eachOutfit: []
+    }
+    this.click = this.click.bind(this);
+    this.delete = this.delete.bind(this);
+  }
+
+
+
+  click() {
+    let eachOutfitcopy = this.state.eachOutfit.slice();
+    let compareId = this.props.overViewStyles.product_id;
+
+    if (eachOutfitcopy.length >= 1) {
+      if (eachOutfitcopy.filter(i => i.id === compareId).length > 0) {
+        return;
+      }
+      eachOutfitcopy.push({
+        id: this.props.overViewStyles.product_id,
+        results: this.props.overViewStyles.results,
+        name: this.props.overViewProd.name,
+        category: this.props.overViewProd.category,
+        description: this.props.overViewProd.description,
+        defaultPrice: this.props.overViewProd.default_price,
+        features: this.props.overViewProd.features
+      });
+      this.setState({
+        eachOutfit: eachOutfitcopy
+      })
+    }
+
+    if (eachOutfitcopy.length < 1) {
+      eachOutfitcopy.push({
+        id: this.props.overViewStyles.product_id,
+        results: this.props.overViewStyles.results,
+        name: this.props.overViewProd.name,
+        category: this.props.overViewProd.category,
+        description: this.props.overViewProd.description,
+        defaultPrice: this.props.overViewProd.default_price,
+        features: this.props.overViewProd.features
+      });
+      this.setState({
+        eachOutfit: eachOutfitcopy
+      })
+    }
+  }
+
+  delete(idNum) {
+    let eachOutfitcopy = this.state.eachOutfit.slice();
+    let indexToDelete = eachOutfitcopy.map((item) => { return item.id; }).indexOf(idNum);
+    eachOutfitcopy.splice(indexToDelete, 1);
+    this.setState({
+      eachOutfit: eachOutfitcopy
+    })
+  }
+
+  render() {
+    if (this.state.eachOutfit.length === 0) {
+      return (
+        <div className="yourOutfitCard">
+          <div onClick={this.click} className="img">
+          CLICK TO ADD TO YOUR OUTFIT
+          </div>
+          <div className="cardRating">*****</div>
+        </div>
+      )
+    }
+    return (
+      <div className="yourOutfitCard">
+        <div onClick={this.click} className="img">
+          CLICK TO ADD TO YOUR OUTFIT
```

Comment:
Just a suggestion, if this piece looks like a button, maybe it could just say "ADD TO YOUR OUTFIT" as the click is implied?